README.zh_CN.md

```diff
@@ -702,12 +703,13 @@ Go 的上下文（context）也是 Go 并发编程的基石之一。上下文允
 
 #### 不理解GC是如何工作的 (#99)
 
-理解如何调优GC能够引发很多收益，例如处理徒增的负载信息更有效率。
+理解如何调优GC能够带来很多收益，例如有助于更高效地处理突增的负载。
+
+#### 不了解Docker或者K8S对运行的Go应用的性能影响 (#100)
 
-#### 不了解在Docker或者K8S中运行应用的性能影响 (#100)
+为了避免CPU throttling（CPU限频）问题，当我们在Docker和Kubernetes部署应用时，要知道Go语言对CFS(完全公平调度器)无感知。
 
-To help avoid CPU throttling when deployed in Docker and Kubernetes, keep in mind that Go isn’t CFS-aware.
-为了避免CPU throttling问题，当我们部署应用在Docker和Kubernetes时，要记住Go语言不支持CFS(完全公平调度器)。
+> ps: 这意味着Go程序可能会消耗超过其分配的CPU配额，导致CPU限频，常用解决方案是导入 github.com/uber-go/automaxprocs。
```

Comment:
Can you please remove this line? Let's please stick to it being a translation.

Reply:
OK, done.